examples/armv4t/gdb/mod.rs

```diff
@@ -80,6 +80,16 @@ impl Target for Emu {
         target::ext::base::BaseOps::SingleThread(self)
     }
 
+    #[inline(always)]
+    fn use_target_description_xml(&self) -> bool {
+        true
+    }
+
+    #[inline(always)]
+    fn use_register_info(&self) -> bool {
+        true
+    }
+
```

Comment:
you can omit these lines - the functions default to `true`, so there's no need to explicitly redefine them here.